### PR TITLE
Sabrina's REL-2659: ITC support for summing GMOS IFU elements

### PR DESCRIPTION
**Pull request by spakzad**
This is an upgrade to the GMOS ITC to support a new IFU analysis method to sum all apertures with a specified radius of the center, e.g.:
   Sum all IFU elements within ____ arcsec of the center

Please note that there is a problem with one line in ITCRequest.scala.  Carlos asked me to submit the pull request anyway.

### CODE REVIEW

File: bundle/edu.gemini.itc/src/main/java/edu/gemini/itc/gmos/Gmos.java [hunk 1]
```diff
@@ -111,6 +111,13 @@ public Gmos(final GmosParameters gp, final ObservationDetails odp, final String
             } else if (getIfuMethod().get() instanceof IfuRadial) {
                 final IfuRadial ifu = (IfuRadial) getIfuMethod().get();
                 _IFU = new IFUComponent(getPrefix(), ifu.minOffset(), ifu.maxOffset());
+            } else if (getIfuMethod().get() instanceof IfuSum) {
+                double num =  ((IfuSum) odp.analysisMethod()).num();
+                Boolean isIfu2 = true;
```
Comment (cquiroz) [inline bundle/edu.gemini.itc/src/main/java/edu/gemini/itc/gmos/Gmos.java]: This code makes `isIfu2` always true. Is that what you want?

Comment (cquiroz) [inline bundle/edu.gemini.itc/src/main/java/edu/gemini/itc/gmos/Gmos.java]: you could call directly `isIfu2` as:

```java
 _IFU = new IFUComponent(getPrefix(), ((IfuSum) odp.analysisMethod()).num(), isIfu2());
```

File: bundle/edu.gemini.itc.web/src/main/scala/edu/gemini/itc/web/ITCRequest.scala [hunk 2]
```diff
@@ -365,6 +365,7 @@ object ITCRequest {
     case "singleIFU"  => IfuSingle(r.intParameter("ifuSkyFibres"), r.doubleParameter("ifuOffset"))
     case "radialIFU"  => IfuRadial(r.intParameter("ifuSkyFibres"), r.doubleParameter("ifuMinOffset"), r.doubleParameter("ifuMaxOffset"))
     case "summedIFU"  => IfuSummed(r.intParameter("ifuSkyFibres"), r.intParameter("ifuNumX"), r.intParameter("ifuNumY"), r.doubleParameter("ifuCenterX"), r.doubleParameter("ifuCenterY"))
+    case "sumIFU"  => IfuSum(r.intParameter("ifuSkyFibres"), r.doubleParameter("ifuNum"), true);
```
Comment (cquiroz) [inline bundle/edu.gemini.itc.web/src/main/scala/edu/gemini/itc/web/ITCRequest.scala]: What do you mean on the last parameter. do you want to read it from the request?

Comment (spakzad) [inline bundle/edu.gemini.itc.web/src/main/scala/edu/gemini/itc/web/ITCRequest.scala]: I guess so.  Or rather, I just don't want it to always be true.

**Pull request merged**